Feat/add english translation

maximakymenko opened the pull request:
I've managed to add `i18n` and `react-i18n` to handle internationalisation. Also I've created two separate files for PL and EN versions and for now simple function to switch between two languages.

I've translated all text strings that were in the app and tested it. Should be good :) 

Of course, this function needs some improvements to handle more than two languages.

konradkalemba commented on src/components/Layout/Desktop.jsx:
```diff
@@ -1,26 +1,31 @@
+        <Map className={useDarkTheme ? "dark-theme" : ""} />
       </Layer>
       <Layer>
+<<<<<<< HEAD
```
I think you left unresolved confict here.

maximakymenko replied:
Thanks, I had a couple of them, so didn't notice this one. Fixed it.

konradkalemba commented:
Works well! 👍 I noticed a typo tho - I think it should be `voivodeships` instead of `voivodships`.

kkostepski commented on src/components/DailyGrowth/DailyGrowth.jsx:
```diff
@@ -1,96 +1,136 @@
   return (
     <StyledCard
       style={$theme => ({
+        width: "320px",
```
By changing this part of the app, you will actually revert this PR https://github.com/konradkalemba/korona.ws/pull/41

kkostepski commented on src/components/DailyGrowth/DailyGrowth.jsx:
```diff
@@ -1,96 +1,136 @@
+        {isLoading ? (
+          <Loader />
+        ) : (
+          <ResponsiveContainer height={180} width={"99%"}>
```
same here

maximakymenko commented:
Thanks for noticing, I’ll fix it later today

konradkalemba commented:
> To be honest I don't like idea of reformatting code "by the way". Most of changes are related to code reformat. I can see that the code is now "prettier", but It caused pull request is very hard to read :) But final decision/opinion is up to owner @konradkalemba

Yes, @kkostepski is right. I didn't notice the formatting changes at first, but you can leave them for now. I'll configure `prettier` tonight to have consistent code formatting across all components.

Make changes to ensure your PR won't reverse any other PRs that @kkostepski pointed out.

maximakymenko commented:
Sorry guys, it happened  because of conflicts. @kkostepski thanks a lot for pointing that out 👍 I've revert changes from last PR and fixed the typo.

konradkalemba commented:
Hey, thanks for fixes. I found two more mistakes:
![image](https://user-images.githubusercontent.com/8682104/77259828-7b663480-6c84-11ea-98f9-244ecdb70caf.png)
![image](https://user-images.githubusercontent.com/8682104/77259853-8f119b00-6c84-11ea-97ea-e6facffcdd9f.png)

maximakymenko commented:
Thanks, didn’t notice as well. Will be fixed soon ;)

kkostepski commented on src/components/Layout/Desktop.jsx:
```diff
@@ -25,21 +27,14 @@ export default function Layout() {
       </Layer>
       <Layer>
-        <Block position={'fixed'} bottom={0} left={0} width={['100%', '100%', 'auto']} margin={['0', '0', '20px']}>
-          <Block
```
This part of code (entire <Block> with override styles should be included as well :P

maximakymenko commented:
Alright guys, now it should be good! I fixed all naming typos and reverted the last changes for responsiveness. 

Thanks again for all your remarks :) and good night! 

@kkostepski  @konradkalemba  thank you guys!

konradkalemba commented:
@maximakymenko Thanks for your contribution. I appreciate it. :)

kkostepski commented:
I have tested it as well. Good job @maximakymenko!
